_layouts/home.html

```diff
@@ -5,6 +5,14 @@
   {% include header_home.html %}
 
   <div class="site-content home" id="site_content" aria-label="Content">
+    {% if page['Announcements Section']['Intro Text'].size > 0 %}
```

Comment:
I take it we don't have much control over the controller or the model here, right?  Ideally, we want to keep the view simple and just have embedded logic like:

```
{% announcement_section %}
```

or something, so we're not riddling the view layer with a lot of conditional logic.

No need to take action here, but just wanted to put the though out there.

Reply:
There is the concept of an `_includes` folder where I can include a section called announcements then put all of that logic in there.  I did that for somethings like `contracts-list` or `project-list`, could do something similar but not sure I care too much.